content/en/docs/collector/deployment/multiple-collectors.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Multiple Collectors
+description: Considerations for single-writer responsibility when deploying multiple collectors in a gateway configuration.
+weight: 3
+---
+
+## Deploying Multiple Collectors
+
+When deploying multiple collectors in a gateway configuration, it's important to
+ensure that all metric data streams have a single writer and a globally unique identity.
+
+
+### The Single-Writer Principle
+
+The Single-Writer Principle refers to employing a single logical writer for a particular resource.
+Concurrent access from multiple applications that modify or report on the same data can lead to data
+loss or, at least, degraded data quality. In gateway collector deployments, applying this principle guards
+against sending inconsistent data to the backend.
+
+### Further Context
+
+All metric data streams within OTLP must have a [single writer](https://opentelemetry.io/docs/specs/otel/metrics/data-model/#single-writer).
+In a system with multiple collectors, the single-writer principle is most relevant for receivers that generate their
+own metrics, such as scraping receivers or the host metrics receiver.
+This is most important for receivers that create their own metrics, such as pull based scraping
+that targets a specific metric source.
+
+### Deployment Considerations
+
+#### Host Metrics Receiver
+
+When creating metrics related to the host system via the [host metrics receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/hostmetricsreceiver),
+it is important to have only one host metrics receiver instance per host.
+A violation of the single-writer principle in this scenario would mean deploying more than one host metrics receiver
+on the same host. If both try to collect system data at the same time, this may result in
+inconsistent data or data loss. Collisions resulting from inconsistent timestamps may lead to an unstable or inconsistent
+representation of metrics, such as CPU usage.
+
+### Scaling considerations
```

Comment:
This selection doesn't seem like it would belong in `deployment` I don't think, actually.

I'm wondering if it's even necessary to mention. 

This is information basically already covered in `Scaling the Collector` and I'm wondering if it's redundant, or there should be some additional mention in that document about the single-writer principle there.

Reply:
Scaling collectors is another way of saying that we need multiple collectors, so, this page is bound to repeat some/much of what's in the scaling page. How about we just add a "single-writer principle" to that one instead?